File: gang/week4/각도기.java
class Solution {
    public int solution(int angle) {
        int answer = 0;
        if(angle <= 90) {   //각도가 90보다 작거나 같을 때
            if(angle == 90) {   //90이면 2
                answer = 2;
            }
            else {      //90보다 작으면 1
                answer = 1;
            }
        }
        else {      //각도가 90보다 클 때
            if(angle == 180) {  //180이면 4
                answer = 4;
            }
            else {      //아니면 3
                answer = 3;
            }
        }
        return answer;
    }
}

File: yeongseong/4week/각도기.js
function solution(angle) {
    var answer = 0;
    if(angle < 90){
        answer = 1; // 90 보다 작으면 1
    } else if (angle === 90){
        answer = 2; // 90 과 같다면 2
    } else if(90 < angle  && angle < 180){
        answer = 3; // 98 보다 크고 180 보다 작으면 3
    } else  if(angle === 180){
        answer = 4; // 180 과 같으면 4
    }
    return answer;
}
